Source/Core/Math.js

```diff
@@ -252,6 +252,18 @@ define([
         return CesiumMath.clamp(value, 0.0, rangeMax) / rangeMax * 2.0 - 1.0;
     };
 
+    /**
+     * Converts a scalar value in the range [rangeMin, rangeMax] to a scalar in the range [0.0, 1.0]
+     * @param {Number} value The scalar value in the range [rangeMin, rangeMax]
+     * @param {Number} rangeMin The minimum value in the mapped range.
+     * @param {Number} rangeMax The maximum value in the mapped range.
+     * @returns {Number} A scalar value, where rangeMin maps to 0.0 and rangeMax maps to 1.0.
+     */
+    CesiumMath.normalize = function(value, rangeMin, rangeMax) {
```

Comment:
As a rule, we don't abbreviate names that are part of the public API.  `rangeMax` and `rangeMin` should be `rangeMinimum `and `rangeMaximum`